### PR TITLE
feat: describe application and dashboards

### PR DESCRIPTION
**Pull request by Yves-Jacq**
Add new content and new pages to document the BPI application and dashboards.

### CODE REVIEW

File: modules/application/pages/configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+:description: Explain how to use and configure the CLI to export data from a Bonita database
+
+== Overview
+The Bonita Process Insights application allows you to manage your datasets from your Bonita usage.
+
+You can easily analyze all execution data of your processes to better understand your teams' activities.
+
+It is also possible to set time-based goals (Business Goals) to verify the achievement of your business objectives.
+
+image::/images/BPI_homepage.png[]
```
Comment (benjaminParisel) [inline modules/application/pages/configuration.adoc]:
```suggestion
image::BPI_homepage.png[home page]
```

File: modules/application/pages/configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+== Manage applications
+
+You can add an application if it's the first time you access BPI application or if you want to add a new type of dataset.
+
+image::/images/create application[]
```
Comment (benjaminParisel) [inline modules/application/pages/configuration.adoc]:
```suggestion
image::create application.png[Craete application]
```

File: modules/dashboards/pages/index.adoc [hunk 3]
```diff
@@ -5,3 +5,16 @@
...
+The Overview Dashboard is a powerful tool for gaining deep visibility into business processes, monitoring performance and driving continuous improvement initiatives.
+
+Whether you're a process owner, analyst, or executive, this dashboard equips you with the insights you need to optimize processes and achieve operational excellence.
+
+image::/images/dashboard_overview.png[]
+
+=== Dashboard compare
+The dashboard is a comparison between two processes(different version or same version with different filters) within a BPM (Business Process Management) context.
+
+image::/images/dashboard_process_compare.png[]
```
Comment (benjaminParisel) [inline modules/dashboards/pages/index.adoc]:
```suggestion
image::dashboard_process_compare.png[Process compare dashboard]
```

File: modules/dashboards/pages/index.adoc [hunk 4]
```diff
@@ -5,3 +5,16 @@
...
 
 IMPORTANT: this is a work in progress!!
+
+
+=== Dashboard overview
+The Overview Dashboard is a powerful tool for gaining deep visibility into business processes, monitoring performance and driving continuous improvement initiatives.
+
+Whether you're a process owner, analyst, or executive, this dashboard equips you with the insights you need to optimize processes and achieve operational excellence.
+
+image::/images/dashboard_overview.png[]
```
Comment (benjaminParisel) [inline modules/dashboards/pages/index.adoc]:
```suggestion
image::dashboard_overview.png[Dashboard overview]
```

File: modules/application/taxonomy.adoc
Comment (tbouffard) [inline modules/application/taxonomy.adoc:1]: **nitpick:** This file must be referenced in the root `antora.yml` file to display its content in the global taxonomy located at the left of the page. See https://github.com/Yves-Jacq/bonita-process-insights-doc/blob/bpi_doc_yves/antora.yml
The order of the files in the `nav` declaration matters.

File: modules/application/pages/configuration.adoc
Comment (tbouffard) [inline modules/application/pages/configuration.adoc:1]: **note:** about the "application" module name (and other module names)
This is something that I setup when I initialized the repository. I introduced modules to provide guidance for content organization (split content by topic to ease content update). The module name is part of the URL.
Renaming the modules prior putting the content in production has no impact. If we do it after putting it in production, we will have to configure redirects (with page-alias), so it is probably the right time to think about module naming.

File: modules/dashboards/pages/process-overview.adoc [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+* Conduct a root cause analysis for variants failing temporal objectives, such as the 'Update Case Data' variant, to identify and rectify inefficiencies.
+* Standardize and possibly automate parts of the process to reduce the standard deviation in process duration, thereby increasing predictability and consistency.
+* Review and clarify user responsibilities for all flow nodes to ensure accountability and minimize unassigned tasks.
+
+==== Conclusion
+
+The BPM dashboard offers critical insights into the efficiency and timeliness of business process variants.
+By monitoring these metrics against temporal objectives, organizations can target improvements to enhance overall process performance.
+This documentation is now more aligned with the importance of meeting temporal objectives and includes a xref: index.adoc[Glossary] to help clarify technical ssary to help clarify technical terms.
```
Comment (tbouffard) [inline modules/dashboards/pages/process-overview.adoc]: **suggestion:** this xref is broken
```suggestion
This documentation is now more aligned with the importance of meeting temporal objectives and includes a xref:index.adoc[Glossary] to help clarify technical ssary to help clarify technical terms.
```

File: modules/dashboards/taxonomy.adoc [hunk 6]
```diff
@@ -1,2 +1,3 @@
 * Dashboards
-** xref:index.adoc[Introduction]
```
Comment (tbouffard) [inline modules/dashboards/taxonomy.adoc:2]: **nitpick:** this should be restored.
Currently, the index.adoc page is not in the taxonomy whereas it has been updated to include a lot of valuable content.

Comment (tbouffard) [reply modules/dashboards/taxonomy.adoc:2]: fixed in 6abcc43530f4a4929d9051580928d748eddad7a6

File: modules/application/images/BPI_homepage.png
Comment (tbouffard) [inline modules/application/images/BPI_homepage.png:1]: **nitpick:** rename file to `home.png` (enforce kebab case)

File: modules/application/images/create application.png
Comment (tbouffard) [inline modules/application/images/create application.png:1]: **nitpick:** rename file to `create-application.png` (enforce kebab case)

File: modules/application/images/BPI-import-data-page.png
Comment (tbouffard) [inline modules/application/images/BPI-import-data-page.png:1]: **nitpick:** rename file to `import-data.png` (enforce kebab case)

File: modules/dashboards/images/dashboard_overview.png
Comment (tbouffard) [inline modules/dashboards/images/dashboard_overview.png:1]: **nitpick:** rename file to enforce kebab case

File: modules/dashboards/images/dashboard_process_compare.png
Comment (tbouffard) [inline modules/dashboards/images/dashboard_process_compare.png:1]: **nitpick:** rename file to enforce kebab case

**Review by tbouffard — APPROVED** [review]

**Pull request merged**